make/check.mk

```diff
@@ -0,0 +1,5 @@
+##@ Check
+## Targets to be used to ensure code quality
+.PHONY: verify-code
+verify-code: vet lint
```

Comment:
Consider adding a comment so it shows up in the help see https://github.com/Kuadrant/kuadrant-operator/blob/main/make/verify.mk#L17 `make help`

Reply:
So that's how it works? Neat! The more we know. Changed, thanks!